app/livechat/client/views/app/tabbar/visitorEdit.js

```diff
@@ -79,11 +79,15 @@ Template.visitorEdit.onCreated(function() {
 		this.visitor.set(LivechatVisitor.findOne({ _id: Template.currentData().visitorId }));
 	});
 
-	this.autorun(() => {
-		const room = Rooms.findOne({ _id: Template.currentData().roomId });
+	const rid = Template.currentData().roomId;
 
-		this.room.set(room);
-		this.tags.set((room && room.tags) || []);
+	this.autorun(() => {
+		const sub = this.subscribe('livechat:rooms', { _id: rid });
```

Comment:
the `subscribe` can be moved to outside the `autorun`, so only the find needs to be inside the `autorun` to update the reactive vars whenever the room data changes.